dev-lang/janet/janet-0.5.0.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="Dynamic Lisp dialect and bytecode vm"
+DESCRIPTION="Janet is a functional and imperative programming language \
+and bytecode interpreter. It is a modern lisp, but lists are replaced by \
+other data structures with better utility and performance (arrays, tables, \
+structs, tuples). The language also supports bridging to native code written \
+in C, meta-programming with macros, and bytecode assembly."
+HOMEPAGE="https://janet-lang.org/"
+COPYRIGHT="2019 Calvin Rose and contributors"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/janet-lang/janet/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="f5c90122715869657427cbba62809a1d53eaaced2268e7d55202301d65a75076"
+SOURCE_FILENAME="janet-$portVersion.tar.gz"
+PATCHES="janet-0.5.0.patch"
```

Comment:
$portVersion here also

Reply:
But it's already upstreamed

Reply:
you mean there will be a release with these changes?

Reply:
The next release should have it, I think; it's in master.